src/main.rs

```diff
@@ -101,12 +151,83 @@ fn get_sorted_branches(repo: &Repository) -> Result<Vec<String>> {
         .collect();
 
     branch_name_and_commit.sort_by(|(_, a), (_, b)| a.time().partial_cmp(&b.time()).unwrap());
+    branch_name_and_commit.reverse();
 
-    let branches = branch_name_and_commit
-        .iter()
-        .take(20)
-        .map(|(name, _)| name.to_string())
-        .collect();
-
+    let iter = branch_name_and_commit.iter();
+    let branches = match config.limit {
+        Some(limit) => iter.take(limit).map(|(name, _)| name.to_string()).collect(),
+        None => iter.map(|(name, _)| name.to_string()).collect(),
+    };
     Ok(branches)
 }
+
+struct Config {
+    theme: Box<dyn Theme>,
+    fuzzy: bool,
+    limit: Option<usize>,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            theme: Box::new(ColorfulTheme::default()),
+            fuzzy: false,
```

Comment:
It seems to me that even with the `fuzzy` flag enabled you can still select a branch as normal, so we should not even make this as a config option and just enable it by default i think 🤩

Reply:
`fuzzy` flag disables j/k movement which I'm somewhat attached to keeping as an option for those of us that don't like the ergonomics of arrow keys.

Personally, because I'm in the "I'd prefer a good way to search over cleaning up my old branches" camp, I'd set `fuzzy` to `true`, giving up j/k movement for the opportunity to fuzzy-search most of the way, if not all of the way, to the answer.

Reply:
Hmmm, then how about setting fuzzy to `true` by default and keep the option for the hjkl enthusiasts? As much as I enjoy vim bindings I suspect most people would, like you, prefer the search ability 😄